src/coreclr-debug/install.ts

```diff
@@ -183,7 +183,8 @@ export class DebugInstaller
                 "System.ComponentModel.Annotations":  "4.1.0",  
                 "System.ComponentModel.EventBasedAsync":  "4.0.11",
                 "System.Runtime.Serialization.Primitives": "4.1.1",
-                "System.Net.Http":  "4.1.0"
+                "System.Net.Http":  "4.1.0",
+	            "System.Reflection.Emit.Lightweight": "4.0.0"
```

Comment:
nuget.org lists the current version of this as 4.0.1. Does this still work if you use that instead?